src/System.IO.UnmanagedMemoryStream/tests/UmsSecurityTest.cs

```diff
@@ -12,30 +12,25 @@ public class UmsSecurityTests
     public static void ChangePositioViaPointer()
     {
```

Comment:
While your here might as well fix the typo in the test method name and add the 'n' in Position :)